src/scripts/main.js

```diff
@@ -1,5 +1,36 @@
 'use strict';
 
 document.addEventListener('click', e => {
-  // write code here
+  const wall = document.querySelector('.wall');
+  const spider = document.querySelector('.spider');
+  const spiderWidth = spider.offsetWidth;
+  const spiderHeight = spider.offsetHeight;
+  const wallWidth = wall.clientWidth;
+  const wallHeight = wall.clientHeight;
```

Comment:
Відступи між змінними не завадять
```suggestion
  const wall = document.querySelector('.wall');
  const spider = document.querySelector('.spider');
  
  const spiderWidth = spider.offsetWidth;
  const spiderHeight = spider.offsetHeight;
  
  const wallWidth = wall.clientWidth;
  const wallHeight = wall.clientHeight;
```